module5/labs/Week3/Authentication.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Authentication Lab
+---
+
+In today's lab, there are 2 tracks.  You may complete the tracks in any order.
+
+<section class='call-to-action' markdown='1'>
+
+## Track 1: Experiment with Identity Scaffolding
+
+When we initially implemented the Identity scaffolding, we chose to override _all_ of the login related pages in the **Add Identity** Dialog.  Some of these pages are not necessary.  Your task is to experiment with which of these pages you want to include, or not.
+
+For each experiment:
+* Create a new ASP.NET MVC application (you will not need a database).
+* Try a new combination of pages to override at this step of the scaffolding.
+    <img src='/assets/images/module5/week3/AddIdentityDialog.png' style='max-width: 50%;'>
+* Keep track of what combo you are using, and keep notes about how this selection impacted your application!
+
+✅Send an overview of your experiments and their result to your instructors. Be sure to include which files you chose to override for each experiment (a screenshot is great).
+
+</section>
+
+<section class='call-to-action' markdown='1'>
+
+## Track 2: Improve Usability of the Application
+
+The application we built in class today is far from perfect.  Your task is to identify 3 improvements that you will make to the application.
+
+The improvements must be related to authentication in some way, but could also include:
+* Data Validations
+* Logging
+* Refactoring
+
+✅Send a list of improvements you would like to make to your instructors - they will approve, or ask for updates!
+
+After you list of improvements is approved, checkout a new branch and implement your list!
+
+✅Commit and push your improvements branch.  Create a Pull Request and send a link to that PR to your instructors.
+
+</section>
```

Comment:
Oh jeez - the authorization lab didn't get pushed up 🤦‍♀️ It is up now!  Definitely a separate lab.

I agree that this authentication lesson is BIG - I moved the PD session to the Authorization day because that lesson has a lot less going on in it!